jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Build.java

```diff
@@ -144,6 +144,7 @@ public Integer call() {
       JibCli.writeImageJson(commonCliOptions.getImageJsonPath(), jibContainer);
     } catch (Exception ex) {
       JibCli.logTerminatingException(logger, ex, commonCliOptions.isStacktrace());
+      Thread.currentThread().interrupt();
```

Comment:
I think we should interrupt only when catching `InterruptedException`, not for all exceptions.

Reply:
That's a good point. I'll fix this